internal/cli/edit_test.go

```diff
@@ -56,26 +56,26 @@ func TestEditOptions_Validate(t *testing.T) {
 			name:          "invalid - cannot edit events",
 			args:          []string{"event/test-event"},
 			expectError:   true,
-			errorContains: "you cannot edit events",
+			errorContains: "edit not permitted for event",
```

Comment:
I would LOVE to see this become less brittle. I know this isnt your doing but we should not be matching strings for errors as its very brittle. 

```go
const ErrEditNotAllowed = errros.New("edit not permitted for event")
			
wantErr: ErrEditNotAllowed,

if errors.Is(err, ErrEditNotAllowed) { // handle
```